Prepare KEB docs (#1179)
kyma-incubator/compass: docs/kyma-environment-broker/08-04-instance-details.md

> # Check instance details

Reviewer: ```suggestion
# Check SKR instance details
```

> This tutorial shows how to get the instance details.

Reviewer: ```suggestion
This tutorial shows how to get the instance details of your Runtime's instance.
```

## Steps

> 1. Export the instance ID that you set during [provisioning](./08-01-provisioning-kyma-environment.md).

Reviewer: ```suggestion
1. Export the instance ID that you set during [provisioning](./08-01-provisioning-kyma-environment.md):
```

```bash
export INSTANCE_ID={SET_INSTANCE_ID}
```

> **NOTE:** Ensure that the BROKER_URL and INSTANCE_ID environment variables are exported as well before you proceed.

> 2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](./03-05-authorization.md) to verify that provisioning/deprovisioning succeeded.

Reviewer: ```suggestion
2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](./03-05-authorization.md) to verify that provisioning/deprovisioning succeeded:
```

```bash
curl --request GET "https://$BROKER_URL/v2/service_instances/$INSTANCE_ID \
--header 'X-Broker-API-Version: 2.13' \
--header "$AUTHORIZATION_HEADER"
```

A successful call returns the instance details:

```json
{
    "service_id": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
    "plan_id": "4deee563-e5ec-4731-b9b1-53b42d855f0c",
    "dashboard_url": "https://console.{DOMAIN}",
    "parameters": {
        "autoScalerMax": 3,
        "autoScalerMin": 1,
        "components": [],
        "machineType": null,
        "maxSurge": 1,
        "maxUnavailable": 1,
        "name": "test",
        "region": "westeurope",
        "targetSecret": "azrspn-ce-skr-dev-00001",
        "volumeSizeGb": 50,
        "zone": null
    }
}
```

> Fields under the parameters field can differ depends on the provisioning input.

Reviewer: ```suggestion

```

> >**NOTE:** field `dashboard_url` is only available if Runtime was provisioned successfully and compass agent registered runtime in the Director.

Reviewer: ```suggestion

```

> >**NOTE:** field `dashboard_url` is only available if Runtime was provisioned successfully and compass agent registered runtime in the Director.

Reviewer: ```suggestion
>**NOTE:** The **dashboard_url** filed is available only if the Runtime was provisioned successfully and the Runtime Agent registered the Runtime in the Director. Fields under the **parameters** field can differ depending on the provisioning input.
```
